float64 erro_x
float64 erro_y
int8 flag
int8 start_opencv